### PR TITLE
fix: sedge build

### LINKED ISSUE #201: sedge: update to v1.0.0
I can't update `sedge` to v1.0.0 because `go mod vendor` doesn't work correctly with it. I opened the issue about that https://github.com/NethermindEth/sedge/issues/272

### PR DESCRIPTION
**Pull request by selfuryon**
Fixed sedge build, now it works

Close #201

### CODE REVIEW

File: packages/utils/sedge/default.nix [hunk 1]
```diff
@@ -2,29 +2,41 @@
...
-    owner = "NethermindEth";
-    repo = pname;
-    rev = "v${version}";
-    sha256 = "sha256-U48ZM/J/Wj8chQvVB1DuqS3zl/qFMxDzlWS7Y3+U43Y=";
-  };
+    src = fetchFromGitHub {
+      owner = "NethermindEth";
+      repo = pname;
+      rev = "v${version}";
+      sha256 = "sha256-rIBHRiVZT3XXZB8zuFOoX02/g/DhBvIwa+sdis1XSYs=";
```
Comment (aldoborrero) [inline packages/utils/sedge/default.nix:18]: @selfuryon let's change this to `sha` (we will update other packages as well)

File: packages/utils/sedge/default.nix [hunk 2]
```diff
@@ -2,29 +2,41 @@
   bls,
   buildGoModule,
   fetchFromGitHub,
-  lib,
   mcl,
-}:
-buildGoModule rec {
+  mockgen,
+}: let
   pname = "sedge";
```
Comment (aldoborrero) [inline packages/utils/sedge/default.nix:8]: @selfuryon by bringing these two variables, do you think nix-index will be able to update those two without issues?

**Review by aldoborrero — CHANGES_REQUESTED** [review]
@selfuryon just only minor nitpicks but once clarified we can proceed to merge! Good work!

File: packages/utils/sedge/default.nix (hunk 2 above)
Comment (selfuryon) [reply packages/utils/sedge/default.nix:8]: Good catch, I didn't think about that, but I tested it and it works fine!

File: packages/utils/sedge/default.nix (hunk 1 above)
Comment (selfuryon) [reply packages/utils/sedge/default.nix]: done, changed it to `hash`

**Comment by selfuryon** [comment]
Fixed a conflint

**Pull request merged**